src/manager.js

```diff
@@ -100,7 +100,7 @@ class Manager extends EventEmitter {
 
     const workerConfig = {
       name,
-      fetch: () => this.fetch(name, options.batchSize || options.teamSize || 1),
+      fetch: () => this.fetch(name, options.batchSize || options.teamSize || 1, options.includeMetadata || false),
```

Comment:
this line needs the new options

Reply:
My bad, I'm so used to TypeScript catching this sort of thing for me!